api/v1alpha/framework/imagerunner.schema.json

```
@@ -197,6 +197,32 @@
       "required": [
         "name"
       ]
+    },
+    "reporters": {
+      "description": "Supported saucectl reporters.",
+      "type": "object",
+      "properties": {
+        "json": {
+          "type": "object",
+          "description": "The JSON reporter merges test results from all suites in the JSON format into a single report.",
```

Maybe something generic like this could work as a description:
```suggestion
          "description": "The JSON reporter creates single report of all executed saucectl suites.",
```